docs/index.md

```diff
@@ -8,17 +8,54 @@
 <p>A Text-to-Speech Transformer in TensorFlow 2</p>
 </h2>
 
+<p class="text">All samples are converted with <a href="https://github.com/fatchord/WaveRNN"> WaveRNN </a>vocoder.</p>
 
-### 🎧 Samples from the autoregressive model, converted with [WaveRNN](https://github.com/fatchord/WaveRNN) vocoder
+## 🎧 Forward model samples
+
+<p class="text">President Trump met with other leaders at the Group of twenty conference.</p>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/trump.wav?raw=true" controls preload></audio>
 
 <p class="text">Scientists, at the CERN laboratory, say they have discovered a new particle.</p>
-<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_transformertts/cern_particle.wav?raw=true" controls preload></audio>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/scientists.wav?raw=true" controls preload></audio>
 
 <p class="text">There’s a way to measure the acute emotional intelligence that has never gone out of style.</p>
-<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_transformertts/EQ.wav?raw=true" controls preload></audio>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/EQ.wav?raw=true" controls preload></audio>
+
+<p class="text">The Senate's bill to repeal and replace the Affordable Care-Act is now imperiled.</p>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/senate.wav?raw=true" controls preload></audio>
+
+### Robustness
+
+<p class="text">To deliver interfaces that are significantly better suited to create and process RFC eight twenty one , RFC eight twenty two , RFC nine seventy seven , and MIME content.</p>
+
+<p class="text">Autoregressive model</p>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_transformertts/hard.wav?raw=true" controls preload></audio>
+
+<p class="text">Forward model</p>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/hard.wav?raw=true" controls preload></audio>
+
+### Speed control
+<p class="text">For a while the preacher addresses himself to the congregation at large, who listen attentively.</p>
+
+<p class="text">Normal speed</p>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/speed_100.wav?raw=true" controls preload></audio>
+
+<p class="text">25% faster</p>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/speed_125.wav?raw=true" controls preload></audio>
+
+<p class="text">10% slower</p>
+<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/speed_090.wav?raw=true" controls preload></audio>
+
+## 🎧 Autoregressive model samples
```

Comment:
I would remove this part as this is put above in the model samples. This way people can compare both the forward and autoregressive version much faster.